### ISSUE TITLE
Slider component isn't accessible 

### ISSUE THREAD

**Issue by bethcollins92** [canonical-web-and-design/vanilla-framework#4259]
The slider has a couple of problems: 

- It doesn't have an `aria-valuenow` property which means as you navigate through using the keyboard, the current value isn't read out just "greater than..."
- We use `min` and `max` values but the [WCAG guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/#slider_roles_states_props) suggest using `aria-valuemin` and `aria-valuemax`
